README.md

```diff
@@ -76,7 +76,7 @@ iec_source
 ... (clipped) ...
 ```
 
-To interact with the Python dataclasses directly, use:
+To interact with the Python data-classes directly, use:
```

Comment:
This change made me think for a second about what to call `dataclasses.dataclass` in regular old English. I've always just stuck with the spelling `dataclass` because it reflects the name from the standard library.

Looking at the standard library docs, https://docs.python.org/3/library/dataclasses.html they alternate between "dataclass" and "Data Class" - and I'd say 👎 on the camel case version. So let's revert this one back
```suggestion
To interact with the Python dataclasses directly, use:
```